push/src/main/java/org/aerogear/mobile/push/UnifiedPushConfig.java

```diff
@@ -0,0 +1,31 @@
+package org.aerogear.mobile.push;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public final class UnifiedPushConfig {
+
+    private String alias;
+    private List<String> categories;
```

Comment:
I would initialize this to `Collections.emptyList()` so that no check that this is null is needed.